src/plugin.js

```diff
@@ -240,7 +261,11 @@ class VR extends Plugin {
     }
 
     msgs.forEach((msg) => {
-      videojs.log(msg);
+      if (typeof msg === 'string') {
+        videojs.log('VR: ' + msg);
+      } else {
+        videojs.log('VR: ', msg);
+      }
```

Comment:
Can't you just do `videojs.log('VR: ', msg);` without the code fork? Should be the same end result, no?